db/flush_job.cc

```diff
@@ -1064,6 +1068,21 @@ std::unique_ptr<FlushJobInfo> FlushJob::GetFlushJobInfo() const {
   return info;
 }
 
+Env::IOPriority FlushJob::GetRateLimiterPriority(
+    const RateLimiter::OpType op_type) {
+  WriteController* write_controller =
+      versions_->GetColumnFamilySet()->write_controller();
+
+  if (op_type == RateLimiter::OpType::kWrite) {
+    if (write_controller->IsStopped() || write_controller->NeedsDelay()) {
+      return Env::IO_USER;
+    }
+    return Env::IO_HIGH;
+  } else {
+    return Env::IO_USER;
```

Comment:
Will update.

Reply:
@anand1976  https://fburl.com/dapnwp9y indicate the Flush Job read is actually regard as User read. The verification read should be the only Flush read, so I will keep it as is. We can discuss more on this.

Reply:
Good point. The default rate_limiter_priority in ReadOptions is IO_TOTAL, which means disable rate limiting. Since this PR is for the write path, we can defer this to the read path PR and discuss it then.